lib/rouge/lexers/handlebars.rb

```diff
@@ -26,7 +26,15 @@ class Handlebars < TemplateLexer
           push :open_sym
         end
 
-        rule(/(.+?)(?=\\|{{)/m) { delegate parent }
+        rule(/(.+?)(?=\\|{{)/m) do
+          delegate parent
+
+          # if parent state is attr, then we have an html attribute without quotes
+          # pop the parent state to return to the tag state
+          if parent.state?('attr')
+            parent.pop!
+          end
+        end
```

Comment:
This feels awfully fragile. It's very dependent on the order of states in the `HTML` lexer. I wonder if there's another way we can approach this.

Reply:
I know. I did look into it, but did not find anything better. If you have any ideas I will have a look again

Reply:
I tried really hard and thought I almost had it, except there was no way to guarantee we were re-entering the HTML lexer at the right state. So in the end, this may be the best way. 

@jneen Do you have any opinion on checking and popping the parent state like this?

Reply:
@dblessing Any update on this?

Reply:
I had a look at alternative solutions and couldn't come up with anything better. I'd be happy to merge this. Let me know if you think otherwise, @jneen .